week-3/B-array-some/exercise.js

```diff
@@ -6,8 +6,14 @@
   - Do not edit any of the existing code
 */
 
+//The function checkNullValue  will exit if it find an element that has a null value
+function checkNullValue(arrElement) {
+  if (arrElement === null) {
+    process.exit();
+  }
+}
 var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
-
+pairsByIndex.some(checkNullValue);
```

Comment:
This is technically correct but the exercise is asking you to find out whether the array contains any null values, then use that to decide whether to exit the process.

Remember that `.some()` returns a boolean that you can decide whether to exit the process or not.